website/demos/cornell-box.pug

```diff
@@ -0,0 +1,16 @@
+extends demo.pug
+
+
+block canvas
+
+  canvas(id = 'demo-canvas', data-backend = 'auto').img-fluid.w-100
+
+
+block breakdown
+  p This should describe the demo in much more detail, e.g., referring to relevant code snippets and resources
```

Comment:
No placeholders please - actual description or leave it out for now (same for cubescape below).